terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfView.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2013-2018 NTT DATA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package org.terasoluna.tourreservation.app.managereservation;
+
+import java.text.SimpleDateFormat;
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.view.document.AbstractPdfStamperView;
+
+import com.lowagie.text.pdf.AcroFields;
+
+@Component
+public class ReservationReportPdfView extends AbstractPdfStamperView {
+
+    @Value("${reservation.report.pdf.path}")
+    String reservationReportPdfPath;
+
+    @Override
+    protected void mergePdfDocument(Map<String, Object> model,
+            com.lowagie.text.pdf.PdfStamper stamper, HttpServletRequest request,
+            HttpServletResponse response) throws Exception {
+        DownloadPDFOutput downloadPDFOutput = (DownloadPDFOutput) model.get(
+                "downloadPDFOutput");
+
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy/MM/dd");
```

Comment:
- [ ] Good to define as constant field. Because this date format is immutable.

Reply:
16f93bee5def6e3a720c58b164a9b1be7b8dc5e3

I change to define dete format as constrant field in above commit .

Reply:
I confuse... you define instance field.

Reply:
5cdc248f8e121d429d283f3c83e61167e5ce536a

sorry, I fixed in above commit .

Reply:
You should move constant fields above instance fields.

Reply:
17747f4e056831894aeca11f0ae48e78cffc1aa5

I moved constant field above instance field in above commit .